wordpress-mobile/gutenberg-mobile
# Autofiling links from clipboard in Button settings

Fixes #https://github.com/wordpress-mobile/gutenberg-mobile/issues/1957#event-3072715893
Ref to gutenberg: https://github.com/WordPress/gutenberg/pull/20462

To test:

1. Copy link into clipboard
2. Add Button block
3. Open Button settings
4. Notice if link is automatically pasted into Button Link URL field

PR submission checklist:
